### DISCUSSION TITLE
Problem downloading the .exe file

### DISCUSSION THREAD

**Discussion by Acedon95** [audiamus/AaxAudioConverter#44] [Q&A] [answered]
Hi,
is there a way to distribute the .exe installation file through another source? The download speed i get downloading the  .exe file is under 1 B/s and i don't have the patience to wait 2 days for the download to finish. When i clone the whole repository my download speed is fine, so maybe could you add it in there?

**Comment by audiamus** [comment, marked as answer]
A slow or aborted download should be a temporary problem. The GitHub server side may have had an issue, which does happen occasionally. They usually fix that pretty quickly.  The "Releases" feature of a GitHub repository is the designated way to distribute binaries and quite reliable in my experience.
